pages/docs/2_modify/advanced/linking_local_version.md

```diff
@@ -58,3 +57,37 @@ By following these steps, you can effectively manage local changes to the base C
 <div class="note">
 Linking multiple different development versions simultaneously will not work, as running <code>$lerna-linker linkSource</code> will overwrite all previously made links.
 </div>
+
+## Yarn Workspaces
+
+The [workspaces functionality of Yarn](https://yarnpkg.com/features/workspaces) can be used to automatically handle the interlinking process of multiple packages. This approach is already used within the various Comunica monorepositories to manage package interdependencies, and can be extended to link local Comunica packages from a monorepository into another local project without the use of `yarn link`.
+
+<div class="note">
+This approach involves the editing of <code>package.json</code> using local relative paths, as well as probable automated modifications to <code>yarn.lock</code> upon install. Such changes will need to be reverted prior to publishing the target project anywhere.
+</div>
+
+For example, given:
+
+1. a local version of the Comunica base monorepository at `/path/to/comunica`, and
+2. a local project depending on Comunica base packages at `/path/to/project`,
+
+it is possible to add the relative paths to local Comunica packages into `/path/to/project/package.json`:
+
+```json
+{
+   "name": "project",
+   "private": true,
+   "workspaces": [
+      ".",
```

Comment:
This `.` may be confusing.

Maybe we can have two examples; one where the current project is a monorepo and has no other workspaces, and another where the current project is a monorepo with already existing workspaces.

Reply:
I changed the example to be a Comunica feature repository, since that is the most likely use case. Do you think it would be clear enough for most people? I think it is also the most common use case for linking like this.

Reply:
Some people may want to link comunica to a non-monorepo as well. Could you also add this as an example.

Reply:
Done!